runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -155,6 +173,9 @@
 	@Getter(AccessLevel.PACKAGE)
 	private boolean unchargedGlory;
 
+	public Map<String, Integer> agilityMap = new HashMap<>();
+	private ExecutorService executorService = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() * 2);
```

Comment:
allocate a higher number of threads, in places like rev caves this cap can easily be exceeded on lower end computers

Reply:
Got rid of my own executor